src/features/deleteUser/ui.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Button, DeleteForeverIcon } from 'shared';
+import { deleteUserModel } from 'features/deleteUser';
+
+export const DeleteUser: React.FunctionComponent = () => {
+  return (
+    <Button
+      sx={{ width: 1 }}
+      onClick={() => {
+        deleteUserModel.userDeleted();
```

Comment:
already deleted or just requested deletion?

Reply:
the event on which the removal is performed. Event naming - past.  Correctly?

Reply:
Yes, but it describes wrong state. Not deleted yet

Reply:
fixed

Reply:
fixed